### PR TITLE
Addedd Wordpress Only 

### PR DESCRIPTION
**Pull request by belcboo**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.
- [x] Documentation field contains a link to a page with proper explanations on environmental variables, volumes or the docker compose file.

### CODE REVIEW

File: public/v2/apps/wordpress-only.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+  "captainVersion": "2",
+  "documentation": "Taken from https://docs.docker.com/compose/wordpress/. Port mapping removed from WP as it is no longer needed",
+  "displayName": "WordPress",
```
Comment (githubsaturn) [inline public/v2/apps/wordpress-only.json:4]: Change display name to WordPress - No Database or something else more descriptive.

File: public/v2/apps/wordpress-only.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          "WORDPRESS_DB_NAME": "$$cap_db_name",
+          "WORDPRESS_DB_USER": "$$cap_db_user",
+          "WORDPRESS_DB_PASSWORD": "$$cap_db_pass"
+        }
+      }
+    }
+  },
+  "instructions": {
+    "start": "WordPress is an online, open source website creation tool written in PHP. But in non-geek speak, it’s probably the easiest and most powerful blogging and website content management system (or CMS) in existence today.\n\n Enter your WordPress Configuration parameters and click on next. A WordPress container will be created for you. The process will take about a minute for the process to finish. \n\nNote: This app is intended for advanced users who'd like to have a central DB in a single container for multiple WP's.",
+    "end": "Wordpress is deployed and available as $$cap_appname-wordpress . \n\n IMPORTANT: It will take up to 2 minutes for WordPress to be ready. Before that, you might see a 502 error page.\n"
```
Comment (githubsaturn) [inline public/v2/apps/wordpress-only.json:24]: End instruction is wrong. WordPress won't be deployed and available. It should say something to hint the user about the next steps (installing a database and configuring certain env vars).

File: public/v2/apps/wordpress-only.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          "WORDPRESS_DB_HOST": "$$cap_db_host:3306",
+          "WORDPRESS_DB_NAME": "$$cap_db_name",
+          "WORDPRESS_DB_USER": "$$cap_db_user",
+          "WORDPRESS_DB_PASSWORD": "$$cap_db_pass"
+        }
+      }
+    }
+  },
+  "instructions": {
+    "start": "WordPress is an online, open source website creation tool written in PHP. But in non-geek speak, it’s probably the easiest and most powerful blogging and website content management system (or CMS) in existence today.\n\n Enter your WordPress Configuration parameters and click on next. A WordPress container will be created for you. The process will take about a minute for the process to finish. \n\nNote: This app is intended for advanced users who'd like to have a central DB in a single container for multiple WP's.",
```
Comment (githubsaturn) [inline public/v2/apps/wordpress-only.json:23]: Add a bit of note to this re-highlighting the fact that it's without a database.

**Comment by githubsaturn** [comment]
@belcboo please open a new PR for phpmyadmin changes. They don't need to be combined in one PR

**Comment by belcboo** [comment]
The phpmyadmin was a total mistake. It's my first PR and I didn't know those changes were going to appear here. I'm finishing the changes you requested for the WP. 

Sorry.

File: public/v2/apps/wordpress-only.json (hunk 1 above)
Comment (belcboo) [reply public/v2/apps/wordpress-only.json]: I already had a note at the end. but change it to make more emphasis that you need to create the DB manually.

File: public/v2/apps/wordpress-only.json (hunk 3 above)
Comment (belcboo) [reply public/v2/apps/wordpress-only.json]: Changed the description to make emphasis that this needs its own DB and that is intended for advanced users.

**Comment by belcboo** [comment]
Just updated the changes you requested, the goal is to have something similar to Ghost - No Database which allows you to configure the DB manually. That way you don't need to create a Container with MySQL for each WP you need to run. I also don't know if you can exclude the change I made to the phpmyadmin. Was a mistake as I said before.

Thanks.

**Comment by githubsaturn** [comment]
Thanks!

**Pull request merged**